site/src/sections/About/index.js

```diff
@@ -0,0 +1,50 @@
+import React from "react";
+import { AboutStyleWrapper } from "./about.style";
+import Openvector from "../../assets/images/about/Vector.png";
+import Closevector from "../../assets/images/about/vector.png";
+
+const Mainabout = () => {
+  return (
```

Comment:
Wrap the whole component within the `Container` component here.

Reply:
added :tiger: